### PR TITLE
Migration of some 'normal' quotes to TeX style quotes

### LINKED ISSUE #1014: Quotation marks aren't "TeX-style"
Yes, I'm *that* bored :)

Some of the quoted text in the TeX documentation uses "" rather than ``''. I've done a pass at changing this and will submit a PR.

### PR DESCRIPTION
**Pull request by robpieke**
## Description

Changed to TeX-style quotation marks ... cue the excitement!
This resolves #1014 

## Tests

(none, other than rebuilding the docs)

## Checklist:

- [X] I have read the [contribution guidelines](../CONTRIBUTING.md).
- [ ] I have previously submitted a [Contributor License Agreement](http://opensource.imageworks.com/cla/).
- [X] I have updated the documentation, if applicable.
- [ ] I have ensured that the change is tested somewhere in the testsuite (adding new test cases if necessary).
- [ ] My code follows the prevailing code style of this project.

### CODE REVIEW

File: src/doc/languagespec.tex [hunk 1]
```diff
@@ -4683,7 +4683,7 @@ \subsection{Surface material closures}
 \indexapi{microfacet()}
 
 Returns a \colorclosure that represents scattering on the
-surface using some microfacet distribution such as "beckmann" or "ggx" (as
+surface using some microfacet distribution such as ``beckmann'' or ``ggx'' (as
```
Comment (robpieke) [inline src/doc/languagespec.tex]: Or maybe `\cf{"beckmann"}` etc?

File: src/doc/languagespec.tex [hunk 2]
```diff
@@ -4699,7 +4699,7 @@ \subsection{Surface material closures}
 The list of supported distributions will change from renderer to renderer, and
 also the behavior when an unsupported distribution is requested. It is up to the
 implementation to issue an error or to default to a supported one. Also it is
-expected that the alias "default" must work as a distribution in any renderer.
+expected that the alias ``default'' must work as a distribution in any renderer.
```
Comment (robpieke) [inline src/doc/languagespec.tex]: Or maybe `\cf{"default"}`?

File: src/doc/languagespec.tex (hunk 1 above)
Comment (lgritz) [reply src/doc/languagespec.tex]: \qkw{beckmann}

(quoted keyword!)

File: src/doc/languagespec.tex (hunk 2 above)
Comment (lgritz) [reply src/doc/languagespec.tex]: \qkw{default}

**Comment by lgritz** [comment]
Closing because I don't have a CLA. But I will search for this kind of mistake and make the changes independently myself. Thanks for alerting me to the problem.

**Comment by robpieke** [comment]
Sorry re CLA :(

**Comment by lgritz** [comment]
No prob. This one was easy to recreate from scratch.

**Pull request closed without merging**